docker/pypi/wmagent/wmagent-docker-build.sh

```diff
@@ -52,15 +52,15 @@ done
 
 dockerOpts=" --network=host --progress=plain --build-arg WMA_TAG=$WMA_TAG "
 
-docker build $dockerOpts -t wmagent:$WMA_TAG -t wmagent:latest  .
+docker build $dockerOpts -t local/wmagent:$WMA_TAG -t local/wmagent:latest  .
```

Comment:
Please educate me on why prefixing it with `local`?

Reply:
I changed this line based on experience with with `coucdb` container. Because there was an upstream repository called `couchdb` which is actually the official one.  

Same here - it would be good during local builds and tests one to distinguish whether he/she runs from `cmsweb/wmagent` or `local/wmagent`.